payment-manager/src/test/java/com/qpay/paymentmanager/repository/WalletRepositoryTest.java

```diff
@@ -0,0 +1,69 @@
+package com.qpay.paymentmanager.repository;
+
+import com.qpay.paymentmanager.model.entity.wallet.WalletEntity;
+import com.qpay.paymentmanager.model.usertype.UserType;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.context.ActiveProfiles;
+import java.math.BigDecimal;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest()
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@ActiveProfiles(value = "test")
```

Comment:
As I remember you don't need to explicitly set profile to `test` in tests. It runs `test` env by default

Reply:
without it, test fails for reason: missing table [wallets], so i think when you create `application-NAME.yaml` you have to activate profile otherwise it wouldn't use it